src/modules/claim/claim.types.ts

```diff
@@ -41,6 +41,13 @@ export enum RegistrationTypes {
   OnChain = 'RegistrationTypes::OnChain',
 }
 
+export enum RejectionReason {
+  ALREADY_ISSUED = 'Email already associated with DID',
+  MAIL_IS_BLACKLISTED = 'Email belongs to blacklisted domain',
+  CONFIRMATION_LINK_EXPIRED = 'The email confirmation link has expired',
+  VERIFICATION_SEND_ERROR = 'Unable to send verification email to provided email address',
+}
+
```

Comment:
I don't think this is necessary. The issuer/rejector should be able to put whatever they want
```suggestion
```